### PR TITLE
docs: add samples from bigquery/transfer/cloud-client

### PR DESCRIPTION
**Pull request by danoscarmike**
Samples from https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/bigquery/transfer/cloud-client

---
> **IMPORTANT:** When merging, use the **rebase and merge** option to preserve the samples commit history!
---

### CODE REVIEW

**Comment by danoscarmike** [comment]
Samples are being moved from an existing repository ([googlecloudplatform/python-docs-samples](http://github.com/googlecloudplatform/python-docs-samples))

**Comment by danoscarmike** [comment]
Samples being moved from an existing repository

File: synth.py [hunk 1]
```diff
@@ -60,9 +63,17 @@
 # ----------------------------------------------------------------------------
 # Add templated files
 # ----------------------------------------------------------------------------
-templated_files = common.py_library(unit_cov_level=79, cov_level=79, samples_test=True)
+templated_files = common.py_library(cov_level=70)
+templated_files = common.py_library(cov_level=70, samples=True)
```
Comment (plamut) [inline synth.py:67]: Only the second line should be retained, or...?

**Review by plamut — CHANGES_REQUESTED** [review]
Looks good, apart from a seemingly unnecessary line in `synth.py`.

**Comment by plamut** [comment]
@danoscarmike Friendly ping, seems like just a minor change is needed.

File: synth.py (hunk 1 above)
Comment (danoscarmike) [reply synth.py]: Done! Apologies for the delay.

**Review by plamut — APPROVED** [review]
LGTM!

**Pull request merged**